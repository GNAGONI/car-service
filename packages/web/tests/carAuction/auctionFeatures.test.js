import React from 'react'
import renderer from 'react-test-renderer'
import AuctionFeatures from '../../components/carAuction/auctionFeatures'

it('renders correctly', () => {
  const tree = renderer.create(<AuctionFeatures />).toJSON()
  expect(tree).toMatchSnapshot()
})
