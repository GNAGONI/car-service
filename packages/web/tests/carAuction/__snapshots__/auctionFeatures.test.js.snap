// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="container"
>
  <div
    className="row"
  >
    <div
      className="col-12"
    >
      <section
        className="content extend auction-features"
      >
        <div
          className="row"
        >
          <div
            className="col-12 col-md-6 col-lg-4 content-block"
          >
            <div
              className="section-head"
            >
              <h2
                className="text-small"
              >
                Are you a
                 
                <strong>
                  Scrap
                  <br />
                   car dealer
                </strong>
                ?
                 
              </h2>
              <div
                className="text-box"
              >
                <p>
                  Lorem ipsum dolor sit amet, consect adipiscing elitd iam nonummy.
                </p>
              </div>
            </div>
            <ul
              className="tick-list"
            >
              <li>
                Build your business page
              </li>
              <li>
                Access to scrap cars in your area
              </li>
              <li>
                Buy scrap cars at auction
              </li>
              <li>
                Receive customer reviews
              </li>
              <li>
                Bespoke reporting
              </li>
            </ul>
            <div
              className="btn-block"
            >
              <a
                className="btn btn-block btn-primary btn-lg"
                href="#"
              >
                Sign up now
              </a>
            </div>
          </div>
          <div
            className="col-12 col-lg-4 d-none d-lg-flex content-block justify-content-center"
          >
            <div
              className="img-area"
            >
              <picture>
                <source
                  srcSet=",  2x\\""
                />
                <img
                  alt="image description"
                  height="415"
                  src=""
                  width="389"
                />
              </picture>
            </div>
          </div>
          <div
            className="col-12 col-md-6 col-lg-4 content-block"
          >
            <span
              className="bg-model"
            />
            <div
              className="section-head"
            >
              <h2
                className="text-small"
              >
                Are you a
                 
                <strong>
                  garage
                  <br />
                   or dealership
                </strong>
                ?
              </h2>
              <div
                className="text-box"
              >
                <p>
                  Lorem ipsum dolor sit amet, consect adipiscing elitd iam nonummy.
                </p>
              </div>
            </div>
            <ul
              className="tick-list"
            >
              <li>
                Build your business page
              </li>
              <li>
                Buy cars at auction
              </li>
              <li>
                Receive customer reviews
              </li>
              <li>
                Bespoke reporting
              </li>
              <li>
                Then include a register button
              </li>
            </ul>
            <div
              className="btn-block"
            >
              <a
                className="btn btn-block btn-primary btn-lg"
                href="#"
              >
                Sign up now
              </a>
            </div>
          </div>
        </div>
      </section>
    </div>
  </div>
</div>
`;
